server/src/lib/api.ts

```diff
@@ -116,13 +117,39 @@ export default class API {
 
     router.use('/challenge', this.challenge.getRouter());
 
+    router.get('/feature/:locale/:feature', this.getFeatureFlag);
+
     router.use('*', (request: Request, response: Response) => {
       response.sendStatus(404);
     });
 
     return router;
   }
 
+  getFeatureFlag = (
+    { params: { locale, feature } }: Request,
+    response: Response
+  ) => {
+    let featureResult = null;
+
+    try {
+      const featureToken = feature as FeatureToken;
```

Comment:
I don't think you can type the argument signature here because `feature` is part of the `Request` object and its parameters only returns `string: string` key-value pairs.